visual-dotnet/SauceLabs.Visual/Models/OperatingSystem.cs

```diff
@@ -0,0 +1,11 @@
+namespace SauceLabs.Visual.Models
+{
+    public enum OperatingSystem
+    {
+        Android,
```

Comment:
like above - should we set a value for each option to be sure the backend will interpret it always the same way?